subprojects/build-cache/src/main/java/org/gradle/caching/BuildCacheEntryWriter.java

```diff
@@ -26,7 +24,6 @@
  *
  * @since 3.3
  */
-@Incubating
```

Comment:
Shouldn't `getSize()` still be incubating?

Reply:
The build cache team says no ;)

Reply:
Well then, carry on. 🙂